src/System/Terminal/Class.hs

```diff
@@ -66,3 +68,20 @@ class MonadPrinter m => MonadScreen m where
   cursorBackward :: Int -> m ()
   cursorPosition :: Int -> Int -> m ()
   cursorVisible :: Bool -> m ()
+
+instance MonadPrinter IO where
+  putLn              :: IO ()
+  putLn               = SIO.putChar '\n'
+  putChar            :: Char -> IO ()
+  putChar             = SIO.putChar
+  putString          :: String -> IO ()
+  putString           = mapM_ putChar
+  putStringLn        :: String -> IO ()
+  putStringLn s       = putString s >> putLn
+  putText            :: Text.Text -> IO ()
```

Comment:
This should use `Data.Text.IOputStr`. And should filter for control characters first.

Reply:
Can do.